src/components/DataTable/table-cells/TableHeadCell.tsx

```diff
@@ -29,11 +35,17 @@ export const TableHeadCell: React.FC<TableHeadCellProps> = ({
             className={clsx('moonstone-TableHeadCell', className)}
             iconStart={iconStart}
             iconEnd={iconEnd}
-            style={style}
             onClick={onClick}
         >
-            {children}
-            {renderSortIcon({sortDirection, isSorted})}
+            <span className="moonstone-TableHeadCell-content flexRow_nowrap alignCenter">
+                {children}
+                {SortIcon && (
```

Comment:
- By default, we should define a default `sortDirection` value `descending`
- I guess we can use `isSorted` to determine whether the column is sorted.